packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -442,9 +437,6 @@ export class DataStores implements IDisposable {
 			const request: IRequest = { url: id };
 			throw responseToException(create404Response(request), request);
 		}
-
-		this.validateNotTombstoned(context, requestHeaderData);
```

Comment:
Sorry if I already made this comment- looks like deprecated `getRootDataStore` API won't log/throw tombstone anymore, that doesn't seem right.

Reply:
We can remove this function in the next major release so I don't really care about adding it here.

Reply:
Alright, up to you.  Let's be sure to get that removal PR out then sooner than later. < 4 weeks to go until that major release.